uaclient/tests/test_cli_status.py

```diff
@@ -0,0 +1,84 @@
+import mock
+import socket
+
+import pytest
+
+from uaclient.testing.fakes import FakeConfig
+from uaclient import util
+
+from uaclient.cli import action_status
+
+M_PATH = "uaclient.cli."
+
+
+RESPONSE_LIVEPATCH_AVAILABLE = [{"name": "livepatch", "available": True}]
+UNATTACHED_STATUS = """\
+SERVICE       AVAILABLE  DESCRIPTION
+livepatch     yes        Canonical Livepatch service
+
+This machine is not attached to a UA subscription.
+See https://ubuntu.com/advantage
+"""
+
+ATTACHED_STATUS = """\
+SERVICE       ENTITLED  STATUS    DESCRIPTION
+cc-eal        no        —         Common Criteria EAL2 Provisioning Packages
+cis-audit     no        —         Center for Internet Security Audit Tools
+esm-infra     no        —         UA Infra: Extended Security Maintenance
+fips          no        —         NIST-certified FIPS modules
+fips-updates  no        —         Uncertified security updates to FIPS modules
+livepatch     no        —         Canonical Livepatch service
+
+Enable services with: ua enable <service>
+
+                Account: test_account
+           Subscription: test_contract
+            Valid until: n/a
+Technical support level: n/a
+"""
+
+
+@mock.patch(
+    M_PATH + "contract.get_available_resources",
+    return_value=RESPONSE_LIVEPATCH_AVAILABLE,
+)
+@mock.patch(M_PATH + "os.getuid", return_value=0)
+class TestActionStatus:
+    def test_attached(self, m_getuid, m_get_avail_resources, capsys):
+        """Check that root and non-root will emit attached status"""
+        cfg = FakeConfig.for_attached_machine()
+        assert 0 == action_status(mock.MagicMock(), cfg)
+        # capsys already converts colorized non-printable chars to space
+        # Strip non-printables from output
+        printable_stdout = capsys.readouterr()[0].replace(" " * 17, " " * 8)
+        assert ATTACHED_STATUS == printable_stdout
+
+        m_getuid.return_value = 1000
```

Comment:
Weren't you dropping this part?

Reply:
ahh did from unattached, not from the attached test case. Dropping.